client-v2/src/components/FrontsEdit/Collection.tsx

```diff
@@ -0,0 +1,246 @@
+import React from 'react';
```

Comment:
This rather large diff is me moving out the `Collection` to a separate container so that we can use a `mapStateToProp` factory in order to then create the `articleVisibilityDetailSelector` for each collection in a tidier way.

Reply:
Ah, this is nice to see, the Fronts component was very large!